library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss

```diff
@@ -0,0 +1,415 @@
+                    padding: 5px;
+                    height: 100%;
+                    border-right: 1px solid #d3d3d3;
+                    p {
+                        margin: 0;
+                        padding: 0;
+                        font-size: 11px;
+                        color: #747474;
+                    }
+                    .points {
+                        color: #444444;
+                        font-style: italic;
+                    }
+                }
+            }
+
+            .criterion-actions {
+                padding: 10px;
+                display: table-cell;
+                background: #f2f2f4;
+                width: 5%;
+                position: relative;
+            }
+        }
+
+        .criterion-title, .criterion-item-title {
+            font-size: 16px;
+            color: #2f2f2f;
+            font-weight: bold;
+            margin: 0;
+            padding: 0;
+            &.criterion-item-title {
+                font-weight: normal;
+                color: #3a3a3a;
+            }
+        }
+
+        &.grading {
+            .rating-item {
+                cursor: default;
+                &:hover:not(.student) {
+                    background: #D7E1E7;
+                }
+                &.selected {
+                    background: #bed8e7;
+                }
+            }
+            .fine-tune-points {
+                min-width: 75px;
+            }
+            strong.points-display {
+                display: block;
+                padding: 2px;
+                text-align: center;
+                &.strike {
+                    text-decoration: line-through;
+                }
+            }
+        }
+    }
+}
+
+.rubric-totals {
+    &.sakai-rubric-criterion-grading,
+    &.sakai-rubric-criterion-student {
+        text-align: right;
+        padding: 5px 10px;
+    }
+}
+
+sakai-rubric-grading-comment,
+sakai-rubric-student-comment {
+    iron-icon.comment-icon {
+        height: 15px;
+        width: 15px;
+        margin: 5px auto;
+        display: block;
+        &.active {
+            fill: orange;
+        }
+    }
+}
+
+[class*="edit-popover"] {
+    width: 400px;
+    .buttons {
+        text-align: right;
+        .btn {
+            margin: 0;
+            &.cancel {
+                color: #777;
+            }
+        }
+    }
+    .form-group {
+        &.title {
+            display: inline-block;
+            width: 70%;
+        }
+        &.points {
+            display: inline-block;
+            width: 28%;
+        }
+    }
+}
+
+.popover.show-tooltip {
+    display: inline-block;
+    position: absolute;
+    .popover-title{
+        display: flex;
+        justify-content: flex-end;
+        .criterion-title{
+            margin-right: auto;
+            margin-bottom: inherit;
+            align-self: center;
+            font-size: 14px;
+        }
+        .buttons{
+            align-self: center;
+            margin-left: 10px;
```

Comment:
There is a variable called `$standard-spacing` that could be used here.

Reply:
Everything done, thanks for the suggestions @fostersdesign !